scripts/verify_tlc_matrix_contract.py

```
@@ -26,40 +37,98 @@ class TlcEntry:
     config: str
 
 
-def parse_tlc_entries(workflow_text: str) -> list[TlcEntry]:
-    """Parse TLC entries from sequential steps in the tlc job."""
+def _iter_job_blocks(workflow_text: str):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_iter_job_blocks`

(ANN202)